book.rb

```diff
@@ -0,0 +1,14 @@
+require './rental'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, book)
```

Comment:
- [ ] This method for adding rentals should take `person` and `date` as parameters, then the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. See code snippet below for clarifications and kindly fix that. 
`def add_rental(person, date)`
`Rental.new(date, self, person)`
`end`